src/sys/wait.rs

```diff
@@ -46,6 +48,30 @@ libc_bitflags!(
     }
 );
 
+/// Possible child targets of`waitpid()`.
+///
+/// The most common variants for waiting are waiting
```

Comment:
Please wrap these comments a little longer so they match the wrapping elsewhere in the file (usually 79 or 99 characters)